include/adm/private/xml_writer.hpp

```diff
@@ -0,0 +1,20 @@
+#pragma once
```

Comment:
any specific reason `xml_composer.{hpp,cpp}` has been deleted an recreated instead of renamed?

Reply:
No, as GitUp usually just does the right thing I just did not pay attention to it.

Reply:
ok ... changing this to a rename would be better in general, but I think it's not worth the trouble in this case ... ?

Reply:
Yes. Let’s just keep it as it is.